theme/src/resources/templates/header.hbs

```diff
@@ -534,6 +581,47 @@ height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
 		const userTags = window.HelpCenter.user.tags;
 
 		if (Liferay.hasKBPermission(userTags)) {
+			// Display Notification
+
+			const headerNotification = document.querySelector('.header-notification');
```

Comment:
Can we move this `const` right above where it is used?

Reply:
Updated! 93eff19